plugin/julia-neo.vim

```diff
@@ -0,0 +1,18 @@
+function! s:RequireJuliaHost(name)
+  " Julia host arguments
+  let args = ['-e', 'using Neovim; start_host()']
+
```

Comment:
I think that it might be better to just let the user create a separate `julia_master` directory in `rplugin` and add the appropriate call to `remote#host#Register` in `.nvimrc`.